vic/drivers/image/Makefile

```diff
@@ -156,4 +156,4 @@ depend: .depend
 	$(CC) $(CFLAGS) -M $(SRCS) > $@
 
 clean::
-	\rm -f .depend
+	\rm -f .depend
```

Comment:
Can you revert the changes to the image driver Makefile?  They are not part of this PR and I don't think we want to hard code the hydra paths in there like this.